public/wp-content/themes/glific/functions.php

```diff
@@ -158,3 +158,44 @@ function show_more_blogs()
 if (function_exists('acf_add_options_page')) {
 	acf_add_options_page();
 }
+
+function wpb_move_comment_field_to_bottom($fields)
+{
+	$comment_field = $fields['comment'];
+	unset($fields['comment']);
+	$fields['comment'] = $comment_field;
+	return $fields;
+}
+
```

Comment:
Please remove line break for consistency.

Reply:
@Satendra-SR done 👍